macros/CompatBeta.ejs

```diff
@@ -254,8 +251,8 @@ for either a preference flag or a compile flag.
 function writeFlagsNote(supportData, browserId) {
   let output = '';
 
-  const firefoxPrefs = 'To change preferences in Firefox, visit about:config.';
-  const chromePrefs = 'To change preferences in Chrome, visit chrome://flags.';
+  const firefoxPrefs = ' To change preferences in Firefox, visit about:config.';
+  const chromePrefs = ' To change preferences in Chrome, visit chrome://flags.';
```

Comment:
Do we need Edge or any other browser?

Reply:
Probably. We haven't had the instructions before this change either, though. Might be worth to file a follow-up issue about this.